articles/documentation-style-guide.md

```diff
@@ -0,0 +1,331 @@
+- These articles can also be combined into larger conceptual `Manual` articles, providing organizational flexibility.
+- Writing individual operator articles ensures complete coverage of all operators.
+
+Creating an individual operator article requires some additional steps.
+For example, to create an individual operator article for a `PredictPoses` operator that will be included in a "Network Inference" `Manual` article as well as in the automatically generated `Reference` doc:
+
+1) Create the `Bonsai_Sleap_PredictPoses.md` article and place it in the `docs/apidoc` folder. To utilize the `overwrite` function, in the markdown file, assign a UID that follows the namespace.operator format. 
+
+```yml
+---
+uid: Bonsai.Sleap.PredictPoses
+---
+Write content here.
+```
+
+2) Create a `Network-Inference.md` article and place it in the `docs/articles` folder. In the markdown file, include a reference to the individual operator.md file.
+
+```markdown
+Other 
+[!include[Title](../apidoc/Bonsai_Sleap_PredictPoses.md)]
+```
+> [!NOTE]  
+> The title is optional.
+
+### Reference section
+
+As mentioned the `Reference` section of the website is auto-generated from [XML documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags) in the code. A unique feature we have integrated into the Bonsai Editor is that when users when users right click on individual operators they will be shown the `Reference` page for that operator. For many users, this might be the most obvious entry point to the package documentation. Thus it is essential that developers ensure that:
+
+1) XML documentation comments are clear, simple and helpful.
+2) Where appropriate, supplement additional information (basic workflow, images, long-form content) with an individual operator article using the `overwrite` function detailed in the previous section.
+
+At present, we understand that the default template for the `Reference` page is a little too technically oriented, and somewhat confusing for Bonsai users to navigate. We are in the midst of revamping the template to cater to a broader audience and will update this section when it becomes availiable.
+
+### Tutorials section
+
+For the tutorials section, we suggest creating individual pages for various applications where the package is likely to be used. For each application, guide learners through the workflow step by step, organizing it into separate exercises with clear objectives and visible results. This structure makes the content more accessible and manageable for learners, especially for complex workflows.
+
+#### Submodule
+For packages with extensive tutorials, multimedia and other large files, a separate repository can be created and imported as a submodule.
+
+A tutorial submodule can be added with the following command in the `docs` directory:
+
+```powershell
+git submodule add https://github.com/bonsai-rx/machinelearning-examples
```

Comment:
```suggestion
git submodule add https://github.com/bonsai-rx/package-examples
```

Using a non-existing repo name here to make sure people are forced to change the command line.